include/s3select_oper.h

```diff
@@ -1148,6 +1164,18 @@ class variable : public base_statement
       column_pos = -1;
       var_value.set_nan();
     }
+    else if (reserve_word == s3select_reserved_word::reserve_word_en_t::S3S_TRUE)
```

Comment:
the syntax rule is not set for true/false 
**/reserve_word == s3select_reserved_word::reserve_word_en_t::S3S_TRUE/** 
will not happen

Reply:
Thanks. That was the reason for the failure. I have pushed the synatx rule. Now the test is passing.